pkg/cmd/server/origin/controller.go

```diff
@@ -97,25 +170,81 @@ func (c *MasterConfig) NewOpenshiftControllerInitializers() (map[string]controll
-	ret["deployer"] = deployer.RunController
+	ret["openshift.io/deployer"] = deployer.RunController
 
 	deploymentConfig := controller.DeploymentConfigControllerConfig{Codec: codec}
-	ret["deploymentconfig"] = deploymentConfig.RunController
+	ret["openshift.io/deploymentconfig"] = deploymentConfig.RunController
 
 	deploymentTrigger := controller.DeploymentTriggerControllerConfig{Codec: codec}
-	ret["deploymenttrigger"] = deploymentTrigger.RunController
+	ret["openshift.io/deploymenttrigger"] = deploymentTrigger.RunController
+
+	// initialize other controllers
+	imageTrigger := controller.ImageTriggerControllerConfig{
+		HasBuilderEnabled: c.Options.DisabledFeatures.Has(configapi.FeatureBuilder),
+		// TODO: make these consts in configapi
+		HasDeploymentsEnabled:  c.Options.DisabledFeatures.Has("triggers.image.openshift.io/deployments"),
+		HasDaemonSetsEnabled:   c.Options.DisabledFeatures.Has("triggers.image.openshift.io/daemonsets"),
+		HasStatefulSetsEnabled: c.Options.DisabledFeatures.Has("triggers.image.openshift.io/statefulsets"),
+		HasCronJobsEnabled:     c.Options.DisabledFeatures.Has("triggers.image.openshift.io/cronjobs"),
+	}
+	ret["openshift.io/image-trigger"] = imageTrigger.RunController
+
+	imageImport := controller.ImageImportControllerOptions{
+		MaxScheduledImageImportsPerMinute: c.Options.ImagePolicyConfig.MaxScheduledImageImportsPerMinute,
+		ResyncPeriod:                      10 * time.Minute,
+
+		DisableScheduledImport:                     c.Options.ImagePolicyConfig.DisableScheduledImport,
+		ScheduledImageImportMinimumIntervalSeconds: c.Options.ImagePolicyConfig.ScheduledImageImportMinimumIntervalSeconds,
+	}
+	ret["openshift.io/image-import"] = imageImport.RunController
 
 	templateInstance := controller.TemplateInstanceControllerConfig{}
-	ret["templateinstance"] = templateInstance.RunController
+	ret["openshift.io/templateinstance"] = templateInstance.RunController
+
+	serviceServingCert := controller.ServiceServingCertsControllerOptions{
+		Signer: c.Options.ControllerConfig.ServiceServingCert.Signer,
+	}
+	ret["openshift.io/service-serving-cert"] = serviceServingCert.RunController
 
+	sdnController := controller.SDNControllerConfig{
+		NetworkConfig: c.Options.NetworkConfig,
+	}
+	ret["openshift.io/sdn"] = sdnController.RunController
+
+	// Overrides the upstream "resourcequota" controller
+	ret["resourcequota"] = controller.RunResourceQuotaManager
+
+	clusterQuotaReconciliationController := controller.ClusterQuotaReconciliationControllerConfig{
+		Mapper:                         c.ClusterQuotaMappingController.GetClusterQuotaMapper(),
+		DefaultResyncPeriod:            5 * time.Minute,
+		DefaultReplenishmentSyncPeriod: 12 * time.Hour,
+	}
+	ret["openshift.io/cluster-quota-reconciliation"] = clusterQuotaReconciliationController.RunController
+
+	clusterQuotaMappingController := controller.ClusterQuotaMappingControllerConfig{
+		ClusterQuotaMappingController: c.ClusterQuotaMappingController,
```

Comment:
@deads2k this code just plumbs the ClusterQuotaMappingController down from the kube master config as far as I remember, so technically this is not a change, just moving part.

@enj can you please help verify this?